core/services/reaper.go

```diff
@@ -22,7 +22,8 @@ func NewStoreReaper(store *store.Store) SleeperTask {
 }
 
 func (sr *storeReaper) Work() {
-	offset := time.Now().Add(-sr.config.ReaperExpiration()).Add(-sr.config.SessionTimeout())
+	offset := time.Now().Add(-sr.config.ReaperExpiration().Duration()).
```

Comment:
Would this be a bit clearer if the two durations were added together than subtracted from time.Now? the two negative durations here feels weird.

Reply:
> Can you explain this "and as subtrahends from time.Times to obtain bounds for identifying old records" ?

Like in `time.Now().Add(-sr.config.ReaperExpiration().Duration())`.

Reply:
Oops, don't know why that comment landed there. Deleting.

Reply:
I created a `Durations.Before(time.Time) time.Time` method, so this became `sr.config.ReaperExpiration().Before(sr.config.SessionTimeout().Before(time.Now()))`